development-defaults.ini

```diff
@@ -151,10 +151,6 @@ no_shirt     = 0
 
 [enums]
 
```

Comment:
I did a grep of the codebase for all cases of ``_RIBBON`` and I didn't find anything other than the ones which are in configspec.ini and I just grepped specifically for ``INDIE_RIBBON`` and ``BAND_RIBBON`` in the code and didn't find anything (there are a couple of unit tests which reference them, but that's it and those tests are broken anyway :()